### ISSUE TITLE
some simple

### ISSUE THREAD

**Issue by ymmmmmy** [superrabbit2023/InvNet#2]

**Comment by ymmmmmy** [comment]
Thank you for publishing your paper on TechRxiv and providing the open-source code. They have been immensely helpful to me. However, I have a few small questions that I would like to ask you. Is the code you provided on GitHub complete? Are the graphs in the paper generated using code to analyze data such as mean squared error? Secondly, I noticed that the performance on the validation set is significantly poor when using pre-training methods. Is this phenomenon normal?
![image](https://github.com/superrabbit2023/InvNet/assets/38836176/78be5deb-1856-4046-b94d-4c21cdbb01c0)

**Comment by superrabbit2023** [comment]
Thanks for your attention.
1. The code is complete.
2. Some of the graphs are generated using MATLAB code and MATLAB machine learning code which has been well explained in the paper.
3. The performance is dependent on what you are comparing with. What we are comparing with is the performance trained from scratch, which is outperformed by transfer learning methods.
